tests/generic_config_updater/test_dynamic_acl.py

```diff
@@ -0,0 +1,1251 @@
+
+    yield all('enabled' in val for val in new_proxy_arp_vals)
+
+    proxy_arp_del_cmd = 'sonic-db-cli CONFIG_DB HDEL "VLAN_INTERFACE|Vlan{}" proxy_arp'
+    for vid, proxy_arp_val in list(old_proxy_arp_vals.items()):
+        if 'enabled' not in proxy_arp_val:
+            # Delete the DB entry instead of using the config command to satisfy check_dut_health_status
+            duthost.shell(proxy_arp_del_cmd.format(vid))
+
+
+@pytest.fixture(scope="module")
+def config_facts(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
+    """Get config facts for the duthost"""
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    return duthost.config_facts(host=duthost.hostname, source="running")['ansible_facts']
+
+
+@pytest.fixture(scope="module")
+def intfs_for_test(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_frontend_asic_index, tbinfo, config_facts):
+    """Get the interfaces that will be used in our ARP test"""
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    asic = duthost.asic_instance(enum_frontend_asic_index)
+    mg_facts = asic.get_extended_minigraph_facts(tbinfo)
+    external_ports = [p for p in list(mg_facts['minigraph_ports'].keys()) if 'BP' not in p]
+    ports = list(sorted(external_ports, key=lambda item: int(item.replace('Ethernet', ''))))
+    po1 = None
+    po2 = None
+
+    is_storage_backend = 'backend' in tbinfo['topo']['name']
+
+    if tbinfo['topo']['type'] == 't0':
+        if is_storage_backend:
+            vlan_sub_intfs = mg_facts['minigraph_vlan_sub_interfaces']
+            intfs_to_t1 = [_['attachto'].split(constants.VLAN_SUB_INTERFACE_SEPARATOR)[0] for _ in vlan_sub_intfs]
+            ports_for_test = [_ for _ in ports if _ not in intfs_to_t1]
+
+            intf1 = ports_for_test[0]
+            intf2 = ports_for_test[1]
+        else:
+            if 'PORTCHANNEL_MEMBER' in config_facts:
+                portchannel_members = []
+                for _, v in list(config_facts['PORTCHANNEL_MEMBER'].items()):
+                    portchannel_members += list(v.keys())
+                ports_for_test = [x for x in ports if x not in portchannel_members]
+            else:
+                ports_for_test = ports
+
+            # Select two interfaces for testing which are not in portchannel
+            intf1 = ports_for_test[0]
+            intf2 = ports_for_test[1]
+
+    logger.info("Selected ints are {0} and {1}".format(intf1, intf2))
+
+    if tbinfo['topo']['type'] == 't1' and is_storage_backend:
+        intf1_indice = mg_facts['minigraph_ptf_indices'][intf1.split(constants.VLAN_SUB_INTERFACE_SEPARATOR)[0]]
+        intf2_indice = mg_facts['minigraph_ptf_indices'][intf2.split(constants.VLAN_SUB_INTERFACE_SEPARATOR)[0]]
+    else:
+        intf1_indice = mg_facts['minigraph_ptf_indices'][intf1]
+        intf2_indice = mg_facts['minigraph_ptf_indices'][intf2]
+
+    asic.config_ip_intf(intf1, "10.10.1.2/28", "add")
```

Comment:
What's this IP used for?

Reply:
not positive, it is from a ARP fixture that was required for building ARP packets, so I ported the majority of the fixture over.

Reply:
I don't think we need the IP. If it's for the ARP test, we only need to add IP from the Vlan subnet to a ptf port. The IP should be like 192.168.0.x.

Reply:
Removed the line entirely, test still seems to pass

Reply:
Replaced IP address with subnet from minigraph_vlan_interfaces